python-package/lightgbm/basic.py

```diff
@@ -591,6 +604,65 @@ def _load_pandas_categorical(file_name=None, model_str=None):
         return None
 
 
+class Sequence(abc.ABC):
+    """
+    Generic data access interface.
+
+    Object should support the following operations:
+
+        # Get total row number.
```

Comment:
I'm not sure whether double `:` was a typo or not (https://github.com/microsoft/LightGBM/pull/4089#discussion_r660289485). But it was acting as code block directive in Sphinx. Now without it we have the following:

![image](https://user-images.githubusercontent.com/25141164/123960871-7b410280-d9b8-11eb-950c-998c61cdd224.png)


I believe that explicitly mark this description as code block will provide a better rendered result:

![image](https://user-images.githubusercontent.com/25141164/123961060-a3c8fc80-d9b8-11eb-981c-da7c59ae67be.png)

cc @shiyu1994 

```suggestion
    Object should support the following operations:

    .. code-block::

        # Get total row number.
```

Reply:
Sorry, my bad. I think your modification is better.

Reply:
I actually tried single `:` and run `make html` but found no change. Guess it's browser cache not refreshing.